zapisy/apps/offer/vote/views.py

```diff
@@ -49,7 +49,7 @@ def vote(request):
     filter_statuses = [ProposalStatus.IN_OFFER, ProposalStatus.IN_VOTE, ProposalStatus.WITHDRAWN]
     qs = Proposal.objects.filter(status__in=filter_statuses).order_by('name')
     proposal_list = []
-    for p in qs.prefetch_related('tags'):
+    for p in qs.prefetch_related('thematic_tags', "specialist_tags"):
```

Comment:
```suggestion
    for p in qs.prefetch_related('thematic_tags', 'specialist_tags'):
```
(i analogicznie w co najmniej jeszcze jednym miejscu)